middlewares/authn.js

```diff
@@ -96,19 +96,27 @@ module.exports = function(req, res, next) {
         return;
     }
 
-    // otherwise look for auth cookies
-    if (req.cookies.pl_authn == null) {
-        // if no authn cookie then redirect to the login page
-        res.cookie('preAuthUrl', req.originalUrl);
-        res.redirect('/pl/login');
-        return;
+    var authnData = null;
+    if (req.cookies.pl_authn) {
+        // if we have a authn cookie then we try and unpack it
+        authnData = csrf.getCheckedData(req.cookies.pl_authn, config.secretKey, {maxAge: 24 * 60 * 60 * 1000});
```

Comment:
```suggestion
        authnData = csrf.getCheckedData(req.cookies.pl_authn, config.secretKey, {maxAge: config.authnCookieMaxAgeMilliseconds});
```
Did I forget to refactor this into `config.authnCookieMaxAgeMilliseconds` or is it meant to be different? (24h vs 30d)

Reply:
I think it was just an oversight. Your change looks good to me.